packages/hpe-icons/package.json

```diff
@@ -39,6 +47,7 @@
     "eslint-plugin-react-refresh": "^0.4.20",
     "globals": "^16.3.0",
     "minimatch": "^10.0.3",
+    "storybook": "^8.6.14",
```

Comment:
I haven't looked into v9 of storybook too much yet but was there a desire to use v8 instead of v9?

Reply:
I initially tried v9, but ran into a bunch of Typescript errors due to some of the other @storybook dev dependencies not being caught up with their v9 + TS support.

Reply:
Ah gotcha, that makes sense